### PR TITLE
Leian/docs

### PR DESCRIPTION
**Pull request by leian7**
Documentation how to use StreamWebs: still in progress. Feedback/suggestions for corrections would be great. 

## Changes in this PR.

- [X] Explains what StreamWebs is in ``about.rst``
- [X] Provides a project outline in ``outline.rst`` (login, resources, and sites)
- [X] Explains how to use StreamWebs in ``usage.rst`` (login, sites)

## Examining this PR.

1. Quick way-- visit the following links: [about.rst](https://github.com/osuosl/streamwebs/blob/leian/docs/docs/source/about.rst), [outline.rst](https://github.com/osuosl/streamwebs/blob/leian/docs/docs/source/outline.rst), [usage.rst](https://github.com/osuosl/streamwebs/blob/leian/docs/docs/source/usage.rst)
2. Alternatively: cd into ``docs``
3. run ``make html``
4. ``cd build``
5. run ``python -m SimpleHTTPServer`` (you may also specify an alternative port, i.e. ``python -m SimpleHTTPServer 8001``, if the default port is already in use)

@Kennric @athai @LyonesGamer

### CODE REVIEW

**Comment by ElijahCaine** [comment]
@leian7 Please remove the `docs/build` directory from this branch and add `docs/build/` to a file named `.gitiginore` at the top level of this repository. We do not track built documentation or binaries with version control.

The command will look something like `git rm --cached -r docs/build` and then commit, but look up removing files in a git repo to make sure you've understand how it's done.

File: docs/source/about.rst [hunk 1]
```diff
@@ -0,0 +1,20 @@
+.. _about:
+
+=====
+About
+=====
+
+StreamWebs is a dynamic networking platform that links students with locally based hands-on  watershed stewardship projects and provides a multimedia showcase for their project and data reports. StreamWebs offers teachers and community partners resources to support setting students on the path to lifelong watershed stewardship. By providing students and teachers open-source, web-based tools for watershed data management, analysis, and networking, StreamWebs supports classrooms in their pursuit of STEM (Science, Technology, Engineering, Mathematics) educational opportunities, while helping students and teachers demonstrate their role as vital contributors to watershed sustainability. StreamWebs originated in 2008 as a project of The Freshwater Trust.  In 2011, StreamWebs transitioned to Oregon State University Extension Service.
```
Comment (ElijahCaine) [inline docs/source/about.rst]: The content here is good.
We tend to wrap our lines at 80 lines in `.rst` source files. Most text editors can do this automatically. If you're using vim you select the line with `shift+v` and run type `gqq` to auto-wrap lines characters (the default line wrap length is 80 characters).

File: docs/source/outline.rst [hunk 2]
```diff
@@ -7,7 +7,7 @@ Project Outline
 Current plan for the StreamWebs rewrite. For a visual representation, 
 refer to `this`_.
 
-For now, the site's homepage will be a static intro that contains a brief
+For now, the site's homepage will be a static intro that contains a brief
```
Comment (ElijahCaine) [inline docs/source/outline.rst]: ~~For now~~

Comment (leian7) [inline docs/source/outline.rst]: @athai wrote the outline. I think the "For now" is there is because it might not be static in the future?

Comment (leian7) [inline docs/source/outline.rst]: Also thanks for the great vim trick!

**Comment by Kennric** [comment]
@leian7 it looks like you have included the user model and test for the index page in this as well - that should be in a separate PR that addresses the index page issue, this one should only contain documentation

**Comment by leian7** [comment]
Yeah, I unfortunately forgot to switch branches when I started working on
code. leian/user_model is where everything is. Should I just go ahead and
remove the extra stuff from leian/docs's model.py and tests.py?

**Comment by leian7** [comment]
@Kennric Thanks, I removed those commits.

**Comment by Kennric** [comment]
It's a good start. +1

**Comment by matthewrsj** [comment]
+1, I think this covers most of what was discussed with the client.

File: streamwebs_frontend/streamwebs/tests.py [hunk 3]
```diff
@@ -1,3 +1,7 @@
 from django.test import TestCase
 
 # Create your tests here.
+class IndexViewTest(TestCase):
```
Comment (matthewrsj) [inline streamwebs_frontend/streamwebs/tests.py]: You should actually remove this, since you add it in #28

Comment (ElijahCaine) [inline streamwebs_frontend/streamwebs/tests.py]: Agreed. Changes in this doc is not in the scope of this PR.

Comment (leian7) [inline streamwebs_frontend/streamwebs/tests.py]: Thanks, it should be gone now.

**Comment by athai** [comment]
Looks good! +1

**Pull request merged**